projects/openthread/radio-receive-done-fuzz.options

```diff
@@ -0,0 +1,2 @@
+[libfuzzer]
+max_len = 127
```

Comment:
Thanks for the suggestion.  Updated this PR and the related one at: https://github.com/openthread/openthread/pull/2014.